src/forms.py

```diff
@@ -1,37 +1,42 @@
 from flask_wtf import FlaskForm
-from wtforms import StringField, PasswordField, TextAreaField, SubmitField
-from wtforms.validators import Email, Length, InputRequired, DataRequired
+from wtforms import StringField, PasswordField, TextAreaField, SubmitField, BooleanField
+from wtforms.validators import Email, Length, InputRequired, DataRequired, EqualTo
+
 
-"""
-[VARIABLE] = [FIELD TYPE]('[LABEL]', [
-    validators=[VALIDATOR TYPE](message='[ERROR MESSAGE]')
-])
-"""
 class RegisterForm(FlaskForm):
     email = StringField('email', [
         InputRequired(),
         Email(message='Invalid email'),
-        Length(max=30, message='Your email is too long')
+        Length(max=60)
```

Comment:
Message not needed anymore?

Reply:
not deleted by me (previous programmer), good that you spotted it